lib/slack-ruby-bot/commands.rb

```diff
@@ -1,5 +1,5 @@
 require 'slack-ruby-bot/commands/base'
 require 'slack-ruby-bot/commands/about'
-require 'slack-ruby-bot/commands/help'
+require 'slack-ruby-bot/commands/help_command'
```

Comment:
This one should be `help`, just like `about` or `hi`, not `_command`, no?

Reply:
Well, I didn't want it to be confusing with `SlackRubyBot::Commands::Help` from `commands/support`. And the class name is `HelpCommand` so I thought its file should match that name.

Reply:
I'm just thinking either all commands are `SomethingCommand` or `Something`. Thoughts?

Reply:
Just `Something` seems to be better but then the help command's class should be renamed to `Help` and we'll have to change the name of `Help` class in support. Or add a namespace like `SlackRubyBot::Commands::Support::Help`.

Reply:
I like the namespace idea.

Reply:
@dblock Okay I'll change it. If there's anything else, let me know. Thanks :)